website/src/reference/builder/member/getter.md

```diff
@@ -0,0 +1,87 @@
+# `overwritable` :microscope:
+
+**Applies to:** <Badge type="warning" text="struct fields"/> <Badge type="warning" text="function arguments"/> <Badge type="warning" text="method arguments"/>
+
+Allows getting a reference to an already set member.
+
+::: danger 🔬 **Experimental**
+
+This attribute is available under the cargo feature `experimental-getter`. Breaking changes may occur between **minor** releases but not between patch releases.
+
+The fate of this feature depends on your feedback in the tracking issue [#149](https://github.com/elastio/bon/issues/221). Please, let us know if you have a use case for this attribute!
+
+:::
+
+::: tip
+
+This attribute is also configurable via the top-level [`#[builder(on(...))]`](../top-level/on)
+
+:::
+
+This attribute now makes the following possible:
+
+```rust
+#[derive(bon::Builder)]
+struct Example {
+    #[builder(getter)] // [!code highlight]
+    x: u32,
+}
+
+let builder = Example::builder().x(1);
+
+let x_ref = builder.get_x(); // [!code highlight]
+    
+builder.build();
+```
+
+This enables builders to hold state that can be accessed by the user while still acting as a builder to some final outcome.
+This is especially user if you have a large builder that is getting passed around to many functions,
+and you need to use some previously set value in the builder to set another value.
```

Comment:
Let's keep the docs as laconic as possible. `bon`'s documentation is already embarassingly big. I think the use case for getters is already clear, so we can just remove this paragraph. Instead, I'd suggest adding an example of getter for an optional member.